index.html

```diff
@@ -0,0 +1,38 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>Pumpkin Dash</title>
+    <link rel="stylesheet" type="text/css" href="style.css"/>
+</head>
+<body>
+<main>
+    <div id="aboveSection" class="topSection">
+        <div class="title"> <h2>Pumpkin Dash</h2></div>
+    </div>
+    <section id="gameBox" class="gameSection">
+        <div id="playScreen" class="gameScreen">
+            <div class="Height30percent"></div>
```

Comment:
class name should be camel case, consistent with other class names
also class name needs to change, try and describe what it will contain, not what it will do where possible